cmd/open.js

```diff
@@ -11,12 +11,36 @@ exports.builder = (yargs) =>
     .strict()
     .usage('Usage: $0 open <package> [options]\n\nOpens the package in your browser..')
     .example('$0 open gulp', 'Opens "gulp" package')
-    .demand(1, 1);
+    .example('$0 open --npms gulp', 'Opens "gulp" package in `https://npms.io` service')
+    .example('$0 open --npm gulp', 'Opens "gulp" package in `https://npmjs.org` service')
+    .example('$0 open --github gulp', 'Opens "gulp" package in `https://github.com` service')
+    .demand(1, 1)
+    .options({
+        npms: {
+            describe: 'Open "gulp" package in `https://npms.io` service',
+            type: 'boolean',
+        },
+        npm: {
+            describe: 'Open "gulp" package in `https://npmjs.org` service',
+            type: 'boolean',
+        },
+        github: {
```

Comment:
Change this option to `auto` because it actually defaults to repository and fallbacks to npm.